Kickstarter-iOS/Views/Controllers/SelectCurrencyViewController.swift

```diff
@@ -24,6 +24,7 @@ final class SelectCurrencyViewController: UIViewController, MessageBannerViewCon
 
     self.tableView.register(UITableViewCell.self, forCellReuseIdentifier: reuseIdentifier)
 
+    self.tableView.setConstrained(headerView: self.headerView)
```

Comment:
Reading this call makes me think we're making `headerView` fill the `tableView`...would it make sense to use a better function signature?

```
func setConstrainedWidth(for headerView: UIView) { }
func setConstrainedToWidth(headerView: UIView) { }
```

Reply:
Sure - I'll improve it!